scapy/utils.py

```diff
@@ -1701,7 +1719,85 @@ def recv(self, size=MTU):
         return self.read_packet()
 
 
-class RawPcapWriter(object):
+class GenericRawPcapWriter(object):
```

Comment:
You could add an unimplemented `_write_header` function to this class (raising NotImplementedError)